app/controllers/auth_source_ldaps_controller.rb

```diff
@@ -47,4 +47,8 @@ def test_connection
     Foreman::Logging.exception("Failed to connect to LDAP server", exception)
     render :json => {:message => exception.message}, :status => :unprocessable_entity
   end
+
+  def controller_permission
+    'authenticators'
```

Comment:
Yeah it's kind of a separate issue, I noticed while testing it that permissions to non-admin users did not apply properly